components/resolved_ts/src/endpoint.rs

```diff
@@ -126,7 +139,37 @@ impl ObserveRegion {
             ResolverStatus::Ready => {
                 for log in change_logs {
                     match log {
-                        ChangeLog::Error(e) => return Err(Error::request(e.clone())),
+                        ChangeLog::Error(e) => {
+                            debug!(
+                                "skip change log error";
+                                "region" => self.meta.id,
+                                "error" => ?e,
+                            );
+                            continue;
+                        }
+                        ChangeLog::Admin(req_type) => match req_type {
+                            AdminCmdType::Split
+                            | AdminCmdType::BatchSplit
+                            | AdminCmdType::PrepareMerge
+                            | AdminCmdType::RollbackMerge
+                            | AdminCmdType::CommitMerge => {
+                                info!(
+                                    "region met split/merge command, stop tracking since key range changed, wait for re-register";
+                                    "req_type" => ?req_type,
+                                );
+                                // Stop tracking read progress so that `tracked_index` larger than the split/merge command index
+                                // won't be published untill `RegionUpdate` event trigger the region re-register and re-scan the
+                                // new key range
+                                self.resolver.stop_tracking_read_progress();
```

Comment:
And resolved ts needs to stop tracking too.

Reply:
Updated, PTAL